exec/java-exec/src/test/java/org/apache/drill/test/RestClientFixture.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.drill.test;
+
+import com.google.common.base.Preconditions;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.apache.drill.exec.server.rest.StatusResources;
+
+import javax.annotation.Nullable;
+
+import static io.restassured.RestAssured.when;
+
+/**
+ * Represents a client for the Drill Rest API.
+ */
+public class RestClientFixture {
+  public static final String REST_API_PORT = "drill.exec.http.port";
```

Comment:
Isn't there already an `ExecConstants` value for this option? See `ExecConstants.HTTP_PORT`.